### PR TITLE
LEAF 2599 tabbing and enter for employee selection

### PR DESCRIPTION
**Pull request by aerinkayne**
Addresses the current inability of users to select orgchart employees from the table dropdowns by tab and enter.
Currently only applies to orgchart employee selection in forms (does not yet apply to report builder)

### CODE REVIEW

**Review by AmandaSumner — APPROVED** [review]
Looks Good

**Review by maxrichardjr — APPROVED** [review]
Looks good - no issues found in Test Automation

**Review by gerald-todd — APPROVED** [review]

**Review by raphaelfretter — APPROVED** [review]

**Review by gsteinhoff — APPROVED** [review]

**Review by gerald-todd — APPROVED** [review]

File: LEAF_Nexus/js/employeeSelector.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ employeeSelector.prototype.showBusy = function() {
 
 employeeSelector.prototype.select = function(id) {
 	this.selection = id;
-
+	if(typeof event.key !== 'undefined' && event.key !== 'Enter') return;
```
Comment (Pelentan) [inline LEAF_Nexus/js/employeeSelector.js:70]: Lets get into the mindset of always "cleaning" data before doing comparisons.  In this case, make event.key all lower case and compare to "enter".

File: LEAF_Nexus/js/nationalEmployeeSelector.js [hunk 2]
```diff
@@ -103,13 +103,13 @@ nationalEmployeeSelector.prototype.showNotBusy = function() {
...
 	$('#' + this.prefixID + 'iconBusy').css('display', 'inline');
-    $('#' + this.prefixID + 'status').text('Loading');
+	$('#' + this.prefixID + 'status').text('Loading');
 	this.isBusy = 1;
 };
 
 nationalEmployeeSelector.prototype.select = function(id) {
 	this.selection = id;
-
+	if(typeof event.key !== 'undefined' && event.key !== "Enter") return; //for keypress events
```
Comment (Pelentan) [inline LEAF_Nexus/js/nationalEmployeeSelector.js:112]: And here

**Review by Pelentan — CHANGES_REQUESTED** [review]

**Review by Pelentan — APPROVED** [review]

**Pull request merged**